client/src/LiveTelemetry.tsx

```diff
@@ -0,0 +1,161 @@
+// /client/App.js
+import React, { Component } from "react"
+import Col from "react-bootstrap/Col"
+import Row from "react-bootstrap/Row"
+import Map from "./Map"
+import axios from "axios"
+
+function ThreeRow(item1, item2, item3) {
+  return (
+    <Row>
+      <Col>
+        {item1}
+      </Col>
+      <Col>
+        {item2}
+      </Col>
+      <Col>
+        {item3}
+      </Col>
+    </Row>
+  )
+}
+
+interface TelemetryData {
+  label : string
+  value : string
+}
+
+function telemetryRow(data1: TelemetryData, data2: TelemetryData, data3: TelemetryData) {
+  return (
+    <React.Fragment>
+      { ThreeRow(<strong>{data1.label}</strong>, <strong>{data2.label}</strong>, <strong>{data3.label}</strong>) }
+      { ThreeRow(data1.value, data2.value, data3.value) }
+    </React.Fragment>
+  )
+}
+
+
+class LiveTelemetry extends Component {
+  state = {
+    speed: 0,
+    voltage: 0,
+    duration: 0,
+    temperature: 0,
+    stateOfCharge: 0,
+    consumption: 0,
+    panelPower: 0,
+    heading: 90,
+    carLocation: {
+      lat: 29.651979,
+      lng: -82.32502,
+    },
+    loading: true,
+  };
+
+  componentDidMount() {
+    setInterval(this.getDataFromDb, 100);
+  }
+
+  getDataFromDb = () => {
+    axios.get("/api/live/data").then((res) => {
+      // var {voltage, gps} = res.data
+      // if (voltage[0] && gps[0]) {
+      //     this.setState({
+      //         voltage: voltage[0].Voltage,
+      //         heading: gps[0].heading,
+      //         speed: gps[0].speed,
+      //         carLocation: {
+      //             lat: parseFloat(gps[0].coordinates.latitude),
+      //             lng: parseFloat(gps[0].coordinates.longitude)
+      //         },
+      //         loading: false
+      //     })
+      // }
+      // else {
+      this.setState({ loading: false });
+      // }
+    });
+  };
+
+  render() {
+    const {
+      // speed,
+      // voltage,
+      // duration,
+      // temperature,
+      // stateOfCharge,
+      // consumption,
+      // panelPower,
+      carLocation,
+      heading,
+      loading,
+    } = this.state;
+
+    if (loading) return <p>Loading....</p>;
```

Comment:
Not super important but maybe add a [spinner](https://react-bootstrap.github.io/components/spinners/) for coolness factor?

Reply:
ooooo nice, I didn't know that was part of the react-bootstrap lib!